src/main/resources/application.yml

```diff
@@ -62,7 +62,7 @@ pba:
 docmosis:
   service:
     uri: ${PDF_SERVICE_BASEURL:https://docmosis.aat.platform.hmcts.net}
-    accessKey: ${PDF_SERVICE_ACCESS_KEY:ZDYxMTkzZTQtMGY2Mi00NDM1LWIyN2ItNGRkNzdjOTczMjAwOjQ1NTE0ODQ}
+    accessKey: ${PDF_SERVICE_ACCESS_KEY:dyVv8pXwQ03RRyJZQIPX2RWP9LgJJGTU08kc9dA8ATJoA9EZXQEWe7L1Uwe}
```

Comment:
this is read from the secrets, do we still need this key here ? should be ideally changed to something dummy

Reply:
if we want to remove this for local, we should do on another ticket